main/res/values/strings.xml

```diff
@@ -1833,6 +1833,7 @@
         <item quantity="many">Add this cache to favorites (%d remaining)</item>
         <item quantity="other">Add this cache to favorites (%d remaining)</item>
     </plurals>
+    <string name="command_set_cache_icons_progress">Setting cache icons</string>
```

Comment:
Maybe: "Set custom cache icons"
I would also suggest this for the cache menu: "Set custom cache icon"

Reply:
On the other hand some translation (e.g. german=benutzerdefiniert) would then be rather long...mmh. 
Unsure..

Reply:
or "Set own cache icons", or ...personal... ?